src/components/table/JadeTable.jsx

```diff
@@ -32,6 +34,9 @@ const styles = (theme) => ({
   spinner: {
     margin: 'auto',
   },
+  cell: {
+    ...ellipsis,
+  },
```

Comment:
is the full set of values in a popover?

Reply:
I don't know if it actually even causes ellipsis but is seems to give the table a good hint to not have such a wonky layout on large values